org.otter_browser.OtterBrowser.json

```diff
@@ -0,0 +1,36 @@
+{
+  "app-id": "org.otter_browser.OtterBrowser",
+  "base": "io.qt.qtwebkit.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "otter-browser",
+  "rename-desktop-file": "otter-browser.desktop",
+  "rename-appdata-file": "otter-browser.appdata.xml",
+  "rename-icon": "otter-browser",
+  "finish-args": [
+    "--device=dri",
```

Comment:
No `xdg-download`?

Reply:
Doesn’t help, still downloads things to /tmp. If downloading via ’Save as…’ then it produces an invalid file. Isn’t there a way to disable portals altogether?

Reply:
> Isn’t there a way to disable portals altogether? 

That is moving in the wrong direction, portals should be used or fixed if there are problems. Disabling them would just be a bad workaround.

Reply:
Or it should be patched to properly download files where they belong;